mitosheet/mitosheet/tests/test_utils.py

```diff
@@ -726,6 +726,32 @@ def checkout_step_by_idx(self, index: int) -> bool:
         )
 
 
+    def generate_graph(
+        self, 
+        graph_preprocessing: Dict[str, Any],
```

Comment:
Does this have the same interface as the function in the API file? It would be nice if it did - which I don't think it does currently. 

I want to be able to pass nice params to this - like graph type, column headers, etc - and have it do the processing necessary to generate the graph!

Reply:
I think I'm confused on how you want to handle these parameters / make it easy for us to pass things straight through. 

This is the same as the API in the api.tsx file
```
  def generate_graph(
        self, 
        graph_preprocessing: Dict[str, Any],
        graph_creation: Dict[str, Any],
        graph_styling: Dict[str, Any],
        graph_rendering: Dict[str, Any],
        step_id: str=None
    ) -> bool:
```

But this is not what you say here: graph type, column headers, etc

Can you clarify which version of this you are looking for. I think I originally had the version where I passed graph type, column headers, etc. but then I switched to this approach after our conversation about the previous PR. So just lmk which one you want and happy to update it to that. 

It looks like the only place where we might be able to pass the params directly through is the graph_styling part which we haven't implemented yet. None of the other places we've done that so far.